.github/workflows/release_linux_aarch64.yml

```diff
@@ -0,0 +1,137 @@
+name: LinuxRelease_aarch64
+
+on:
+  schedule:
+    # Run weekly on Monday 00:00
+    - cron:  '00 00 * * MON'
+  push:
+    branches: [master, rel-*]
+  pull_request:
+    branches: [rel-*, master]  # TODO remove master before merge
```

Comment:
nit: this #TODO comment can be removed now

Reply:
Removed.